datasciencebr/rosie
# Refactor irregular companies tests

**What is the purpose of this Pull Request?**

Extending #108 this PR addresses #106: a refactor of the irregular companies classifier to let it clearer.

**What was done to achieve this purpose?**

Basically using an code architecture that reflects the abstractions used for this case, that is to say, avoid sequences without meaningful labels, organizing data structure with types that allow meaningful attribution and employment of objects and variables…

**How to test if it really works?**

Just run the tests and this should work fine; playing around to see tests failing when they should is a good option too.

**Who can help reviewing it?**

@anaschwendler
